.github/ISSUE_TEMPLATE/copy-edit.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Copy Edit
+about: Issue with spelling, grammar, or clarify of writing
+title: "[COPY EDIT]"
+labels: copy-edit
+assignees: ''
+
+---
+
+## Permalink
+
+(Which page is the problem on? Copy the permalink to the closes subheading *above* the issue by clicking on the ¶ symbol next to that heading.)
```

Comment:
Do you mean closest?

Reply:
Yep! I'll fix this and merge. We can always update once we get going on the process, if needed.